src/components/Result.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Button, Jumbotron } from 'react-bootstrap';
+
+const Result = ({ score, numOfQuestions }) => {
+  return (
+    <div>
+      <Jumbotron>
+        <h2>
+          You have got {score} out of {numOfQuestions}
+        </h2>
+      </Jumbotron>
+      <Button variant="secondary" href="quiz">
```

Comment:
This button still needs a margin because without it the `Review answers` button is right next to it without a gap. It's a good idea to check your changes on the UI before pushing to catch errors like this

Reply:
I checked but I thought both of them had to go, my bad :)